docs/DeveloperGuide.md

```diff
@@ -26,8 +26,9 @@ enhancements.
   * [3.9. Bus at bus stop finder (`/bus` Feature)](#39-bus-at-bus-stop-finder-bus-feature)
   * [3.10. Performing similarity checks](#310-performing-similarity-checks)
   * [3.11. Displaying most searched bus stop](#311-displaying-most-searched-bus-stop-on-start-up)
-    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies)
-- [4. Appendix A: Product scope](#4-appendix-a-product-scope)
+    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies-of-bus-stops-reset-feature)
+  * [3.12. Removing specific delete command (`/deletefav` Feature)](#312-removing-specific-delete-command-deletefav-feature)
+- [4. Appendix A: Requirements](#4-appendix-a-requirements)
```

Comment:
```suggestion
- [4. Appendix A: Product scope](#4-appendix-a-product-scope)
```
Sorry! I just saw the CS2113T website and it says this.

Reply:
No worries! Changed!